keps/prod-readiness/sig-node/3619.yaml

```diff
@@ -4,3 +4,5 @@
 kep-number: 3619
 alpha:
   approver: "@johnbelamaric"
+beta:
+  approver: "@johnbelamaric"
```

Comment:
@johnbelamaric Would you mind reviewing for Production Readiness for Beta of this KEP 🙇 

If you're not available, please let me know. I will look for another prr approver.